pkg/installmanager/installmanager.go

```diff
@@ -231,19 +233,37 @@ func (m *InstallManager) Run() error {
 		os.Exit(0)
 	}
 
-	// will be non-nil if we've been unable to setup SSH, which is a non-fatal error:
-	var sshAgentSetupErr error
+	// sshKeyPaths will contain paths to all ssh keys in use
+	var sshKeyPaths []string
 
-	var sshKeyPath string
-	sshKeyPath, sshAgentSetupErr = m.initSSHKey()
+	// setup the host ssh key if possible
+	sshKeyPath, sshAgentSetupErr := m.initSSHKey(sshPrivKeyPathEnvVar)
 	if sshAgentSetupErr != nil {
 		// Not a fatal error.
-		m.log.WithError(sshAgentSetupErr).Info("unable to initialize ssh agent")
+		m.log.WithError(sshAgentSetupErr).Info("unable to initialize host ssh key")
 	} else {
-		sshCleanupFunc, sshAgentSetupErr := m.initSSHAgent(sshKeyPath)
+		sshKeyPaths = append(sshKeyPaths, sshKeyPath)
+	}
+
+	// setup the bare metal provisioning libvirt ssh key if possible
+	if cd.Spec.Platform.BareMetal != nil {
+		libvirtSSHKeyPath, libvirtSSHAgentSetupErr := m.initSSHKey(libvirtSSHPrivKeyPathEnvVar)
+		if libvirtSSHAgentSetupErr != nil {
+			// Not a fatal error.
+			m.log.WithError(libvirtSSHAgentSetupErr).Info("unable to initialize host ssh key")
```

Comment:
Can we make this info message distinguishable from the one for the other ssh key? They both say "host ssh key".